swift/Sources/FlatBuffers/ByteBuffer.swift

```diff
@@ -387,6 +387,16 @@ public struct ByteBuffer {
       removing: _writerSize &- removeBytes)
   }
 
+  /// Returns the written bytes into the ``ByteBuffer``
+  public func toArray() -> [UInt8] {
```

Comment:
nit: can we keep it a computed property instead of a function? we can called it `underlyingBytes`